soccer/gameplay/plays/restarts/basic_direct.py

```diff
@@ -0,0 +1,57 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import evaluation
+import tactics.coordinated_pass
+import skills
+
+
+class BasicDirect(standard_play.StandardPlay):
+
+    #Minimum probability for which a shot should be taken
+    MIN_SHOT_PROB = 0.35
+    FIELD_LENGTH_MULTIPLIER = 0.75
+
+    class State(enum.Enum):
+        kicking = 1
+        passing = 2
+
+    def __init__ (self):
+        super().__init__(continuous = True)
+        for s in DirectFreeKick.State:
+            self.add_state(s, behavior.Behavior.State.running)
+        # If chance of scoring is greater than equal to MIN_SHOT_PROB it will kick
+        self.add_transition(behavior.Behavior.State.start,DirectFreeKick.State.kicking,
+            lambda:evaluation.shooting.eval_shot(main.ball().pos, main.our_robots()) >= MIN_SHOT_PROB,'ShotChanceHigh')        
```

Comment:
MIN_SHOT_PROB undefined. Either use BasicDirect.MIN_SHOT_PROB or make it an instance variable, same for FIELD_LENGTH_MULTIPLIER